frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -157,4 +157,38 @@ function normalizeConstraintName(constraint: string): ForeignKeyConstraint {
   }
 }
 
+function convertToPostgresColumnType(
+  type: string,
+  enums: readonly DMMF.DatamodelEnum[],
+): string {
+  const enumNames = enums.map((e) => e.name)
+
+  if (enumNames.includes(type)) {
+    return 'enum'
+  }
```

Comment:
In current postgres parser also display real enum value (e.g.  `AnnotationQueueStatus` ).
Need to figure out how to display of enum, I think we can delete this line now and display it as is. What do you think?

Reply:
I see your point. I had been under the impression that displaying the ``PostgreSQL`` data type was the standard approach. However, I believe that showing the enum's display name is a more user-friendly solution.👍

Reply:
This has been fixed in commit: https://github.com/liam-hq/liam/pull/572/commits/23d43907fde4a8de8744bfec0f16557b758c9f1e